sql/core/v2.3/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilterSuite.scala

```diff
@@ -74,9 +75,9 @@ class OrcFilterSuite extends OrcTest with SharedSparkSession {
   }
 
   protected def checkFilterPredicate
```

Comment:
Seems we don't need to change here

Reply:
Might be changed by IDE. Going to revert it.